docs/GETTING_STARTED.md

```diff
@@ -1,288 +1,247 @@
-# Getting Started With SOCI Snapshotter
-This document is a guide to walk through an example of using the
-soci-snapshotter.  To get everything running we will need to do a few things:
-- prerequisites
-- start a local references-enabled OCI registry
-- push an image to our registry using ctr
-- create and push a SOCI index to our registry
-- run a lazily loaded container from the registry
-
-Along the way we will make sure to inspect each of our actions to ensure that
-they worked.  This is valuable to help learn about the tool and debugging.
+# Getting Started With soci-snapshotter
+
+This document walks through how to use soci-snapshotter, including building SOCI
+index, pushing/pulling an image and associated SOCI index, and running a container
+with soci-snapshotter.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Prerequisites](#prerequisites)
+- [Install soci-snapshotter](#install-soci-snapshotter)
+- [Push an image to your registry](#push-an-image-to-your-registry)
+- [Create and push SOCI index](#create-and-push-soci-index)
+  - [Create SOCI index](#create-soci-index)
+  - [(Optional) Inspect SOCI index and ztoc](#optional-inspect-soci-index-and-ztoc)
+  - [Push SOCI index to registry](#push-soci-index-to-registry)
+- [Run container with soci-snapshotter](#run-container-with-soci-snapshotter)
+  - [Configure containerd](#configure-containerd)
+  - [Start soci-snapshotter](#start-soci-snapshotter)
+  - [Lazily pull image](#lazily-pull-image)
+  - [Run container](#run-container)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 ## Prerequisites
```

Comment:
Since we no longer are using the local registry unfortunately a prereq is an AWS account

Reply:
I feel public registry is a more realistic example than a local registry. A dockerhub registry also works. (i.e. the only difference is to simply change the `REGISTRY` to dockerhub or any registry).